.github/workflows/release.yml

```
@@ -17,13 +17,41 @@ jobs:
+            curl \
+            tzdata \
+            pkg-config \
+            libssl-dev \
+            gnupg2 \
+            software-properties-common \
+            git \
+            libpango1.0-dev \
+            gcc-aarch64-linux-gnu
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)